dashboard/src/pages/TreeDetails/Tabs/Build/BuildTab.tsx

```diff
@@ -156,7 +156,7 @@ const BuildTab = ({ treeDetailsData }: BuildTab): JSX.Element => {
       ) : (
         '-'
       ),
-      date: row.date?.split('T')[0],
+      date: row.date ?? '-',
```

Comment:
same issue here, you are paassing an invalid date that is tricking the typing because it is using a string instead of a valid timestamp  `new Date('-')' will return `Invalid Date`